### ISSUE TITLE
FTC Limelight 3A Stream From Onboard Robot

### ISSUE THREAD

**Issue by icemannie** [LimelightVision/limelight-feedback#22]
Debugging vision on an FTC robot is hard when the stream and tuning interface isn't available without unplugging and using a laptop.
Is it currently possible to do that?
If not are there plans to implement it?

**Comment by seweed3** [comment]
@icemannie have you found any way to get the limelight 3a to the robot via camera stream. We couldn't find out how to do it too

**Comment by icemannie** [comment]
If your problem is just getting it to connect in Network Tables, see [this](https://www.chiefdelphi.com/t/help-with-limelight-3a-use-for-frc-network-tables/483483)
or others you can find by searching for 3A.

For the specific situation in this thread, for setup we finished up connecting to the robot using a USB cable and port forwarding the LiimeLight3A in code.

Then we could leave the LimeLight3A connected to the robot USB port and access it via 172.22.11.2:5801.
Haven't revisited to try getting a stream on Elastic or other dashboards while using a radio connection.

**Comment by seweed3** [comment]
Yes, I think I found the solution using FTControl in code and they have a library for limelight plugin to view limelight's interface while running the bot
